packages/xrpl/src/models/methods/ledger.ts

```diff
@@ -96,30 +202,57 @@ export interface LedgerBinary
   transactions?: string[]
 }
 
+interface LedgerResponseBase {
+  /** Unique identifying hash of the entire ledger. */
+  ledger_hash: string
+  /** The Ledger Index of this ledger. */
+  ledger_index: number
+  /**
+   * If true, this is a validated ledger version. If omitted or set to false,
+   * this ledger's data is not final.
+   */
+  queue_data?: Array<LedgerQueueData | string>
+  /**
+   * Array of objects describing queued transactions, in the same order as
+   * the queue. If the request specified expand as true, members contain full
+   * representations of the transactions, in either JSON or binary depending
+   * on whether the request specified binary as true.
+   */
+  validated?: boolean
+}
+
+interface LedgerResponseResult extends LedgerResponseBase {
+  /** The complete header data of this {@link Ledger}. */
+  ledger: LedgerBinary
+}
+
+// This is the default
+// expand = false
+// binary = false
+// Matches basic LedgerRequest
```

Comment:
Ditto

Reply:
I left them there because it seemed helpful for devs, but I can remove if you think it looks bad

Reply:
I think I was mostly confused by the styling. It has a different comment style than the rest. 

I'd recommend moving those into the docstring instead of having them be separate. 

For example: 

"This is the default request response, triggered when `expand` and `binary` are both false."

Reply:
Fixed